knowledge-base/migrate-to-the-html5-viewer-from-the-old-asp.net-webforms-reportviewer-control.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrate to the HTML5 Viewer from the old ASP.NET WebForms ReportViewer control
+description: Replace the old ASP.NET WebForms ReportViewer control with the HTML5 Viewer.
+type: how-to
+page_title: Start using the HTML5 Viewer instead of the old ASP.NET WebForms ReportViewer control
+slug: migrate-to-the-html5-viewer-from-the-old-asp.net-webforms-reportviewer-control
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+        <tr>
+			<td>Project Type</td>
+			<td>Web Projects</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>Legacy ASP.NET Report Viewer</td>
+		</tr>
+	</tbody>
+</table>
+
+  
+## Description
+
+> The **old ASP.NET ReportViewer control is obsolete** as of [Telerik Reporting Q3 2015](https://www.telerik.com/support/whats-new/reporting/release-history/telerik-reporting-q3-2015-(version-9-2-15-930)).  
```

Comment:
Remove the "old" word here as if there isn't a new control that is named exactly like the old one, it is not necessary.